src/main/kotlin/com/terraformation/backend/tracking/db/ObservationStore.kt

```diff
@@ -798,18 +800,25 @@ class ObservationStore(
   }
 
   fun removePlotFromTotals(monitoringPlotId: MonitoringPlotId) {
-    val (plantingSiteId, plantingZoneId) =
+    val (plantingSiteId, plantingZoneId, isAdHoc) =
         dslContext
             .select(
-                PLANTING_ZONES.PLANTING_SITE_ID.asNonNullable(), PLANTING_ZONES.ID.asNonNullable())
+                PLANTING_ZONES.PLANTING_SITE_ID.asNonNullable(),
+                PLANTING_ZONES.ID.asNonNullable(),
+                MONITORING_PLOTS.IS_AD_HOC.asNonNullable(),
```

Comment:
The `asNonNullable` on the zone and site IDs can go away since the values are actually nullable now.